### PR TITLE
Change "schema" to "json file"

### PR DESCRIPTION
**Pull request by huber-nicolas**
In my opinion "json file" is clearer here.

### CODE REVIEW

**Comment by albertmink** [comment]
It's JSON, not json.

File: docs/json.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Normally, if an ABAP object is serialized, only the components of the correspond
 is added. Note that also the `$required` annotation leads to such a behavior.
 
 ### Default Values
-To ensure that only components whose value is not equal to a specific default value are written to the schema, the annotation
+To ensure that only components whose value is not equal to a specific default value are written to the json file, the annotation
```
Comment (huber-nicolas) [inline docs/json.md:141]:
```suggestion
To ensure that only components whose value is not equal to a specific default value are written to the JSON file, the annotation
```

File: docs/json.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ To ensure that only components whose value is not equal to a specific default va
 ```abap
 "! $default 'value'
 ```
-Note that if you specify a default value, the initial values are written to the JSON schema, unless they are not equal to the selected default.
+Note that if you specify a default value, the initial values are written to the json file, unless they are not equal to the selected default.
```
Comment (huber-nicolas) [inline docs/json.md:154]:
```suggestion
Note that if you specify a default value, the initial values are written to the JSON file, unless they are not equal to the selected default.
```

**Comment by larshp** [comment]
from a user perspective, they will have json files, which contain the structure defined in the json schema files

referring to the json files here, which are the json schema files, I'd consider misleading, as it is not the json files that the users will typically experience

**Review by wurzka — APPROVED** [review]
LGTM :)

**Comment by albertmink** [comment]
As mentioned before, it is more precise to say JSON Schema than JSON file.
The entire section is dedicated to `How Abap annotations translates to JSON Schema`. So in this context, I prefer that the documentation is precise and states JSON Schema.

**Comment by wurzka** [comment]
But it is described how data is serialised. And serialisation does not result in a JSON schema

**Comment by huber-nicolas** [comment]
> But it is described how data is serialised. And serialisation does not result in a JSON schema For example: If you specify a $default in the schema, the value is written to the JSON file iff the ABAP value does not equal the specified default value. The value is not written to the schema. There is only the "default":... entry

Yes, @wurzka you understood my intention correctly.
Maybe we have to find better names for a JSON schema and JSON Instances (the files in the "Examples" folder), so that they dont get missunderstood.

**Comment by wurzka** [comment]
> Yes, @wurzka you understood my intention correctly. Maybe we have to find better names for a JSON schema and JSON Instances (the files in the "Examples" folder), so that they dont get missunderstood.

This would be the best as both types of files are "JSON files". But I'm not sure how to name the serialised JSON files correctly.

**Comment by albertmink** [comment]
Yeah, the text aims to explain how the annotations translates to JSON Schema **AND** some special behavior/rules for de-/serialization of JSON (data) files. 

We should include also that the annotation `default` translates to `default` in the JSON Schema itself, see https://github.com/SAP/abap-file-formats/blob/main/file-formats/intf/intf-v1.json#L53

**Comment by huber-nicolas** [comment]
My proposal for the names 

1. JSON Schema
2. JSON Data

I looked which names other website use:
json-schema org:
https://json-schema.org/understanding-json-schema/index.html
"**JSON Schema** is a powerful tool for validating the structure of **JSON data**"

wikipedia:
https://en.wikipedia.org/wiki/JSON#Metadata_and_schema
"**JSON Schema** specifies a JSON-based format to define the structure of **JSON data** for validation..."

**Comment by huber-nicolas** [comment]
> But do we write sth. to JSON data (like in "...non-initial value are written to the JSON data") or would it be better to say we write to the JSON data file? Or file with JSON data... Very complicated 😄

I would go for: JSON Data file

**Review by albertmink — APPROVED** [review]
Look's good to me.

**Pull request merged**